models/DataObject/Listing/Dao.php

```diff
@@ -173,7 +173,7 @@ public function getCount()
      */
     public function loadIdList()
     {
-        $query = $this->getQuery();
+        $query = $this->getQuery([new Expression(sprintf('%s as o_id', $this->getTableName() . '.o_id')), 'o_type']);
```

Comment:
This looks odd, array or string ?

![image](https://user-images.githubusercontent.com/6142086/75032214-4eb5d600-54a8-11ea-8a49-efaa5ef273aa.png)

Reply:
@weisswurstkanone well, that comes from here: https://github.com/pimcore/pimcore/blob/934c42b6cb2085cbd96889f542e4b888c4e02689/lib/Db/ZendCompatibility/QueryBuilder.php#L356